### DISCUSSION TITLE
Updates

### DISCUSSION THREAD

**Discussion by rickninja765** [LivingWithHippos/unchained-android#314] [General]
Seeing no updates since 3 months? Can we expect good news this month?

**Comment by LivingWithHippos** [comment]
work is being done on the branch `feat/material-3`

https://github.com/LivingWithHippos/unchained-android/tree/feat/material-3

it is more or less complete so after porting to material 3 I'll add a couple of small fixes and features and publish a new release.

When I'll merge it in the `dev` branch you'll be able to test it without waiting for a release, following the instructions at step 2 of  https://github.com/LivingWithHippos/unchained-android#installation-calling to get the apk from the Actions tab

As always this is a side project that makes zero money and I consider it more or less "completed" so I only work on it when I have enough time

**Reply by LivingWithHippos** [reply]
Whenever I remember I also update the project with what I'm working on

https://github.com/LivingWithHippos/unchained-android/projects/1

**Comment by elpocoburrito** [comment]
Honestly, I just visited again to see if there was an update, but the project is in a 100% working state and it's already a 12/10 app in my book, so the project doesn't need any updates in my opinion, they're just icing on the cake.

Take your time and know that many of us are extremely thankful for this app!
